README.md

```diff
@@ -1,7 +1,36 @@
-# falcon-helm
+# CrowdStrike Falcon Helm Chart and Helm Operator
+
+[Falcon](https://www.crowdstrike.com/) is the [CrowdStrike](https://www.crowdstrike.com/)
+platform purpose-built to stop breaches via a unified set of cloud-delivered
+technologies that prevent all types of attacks — including malware and much
+more.
+
+The CrowdStrike Falcon Helm Chart and the Helm Operator are designed to deploy
+and manage the Falcon sensor on your Kubernetes cluster of choice.
+[Helm Charts](https://helm.sh/docs/topics/charts/Helm) and Helm-based Kubernetes
+Operators are packaging methodologies for Kubernetes. Which methodology you use
+is dependent on what the Kubernetes vendor and implementation supports.
+
+# Deployment
```

Comment:
```suggestion
# Pre-launch Checklist
```

Reply:
Let's make this `Deployment Considerations` instead of `Pre-launch Checklist`.

Reply:
fiiiiiine